pages/learn/deploy/deployment.md

```diff
@@ -25,101 +25,25 @@ Once the cloud builder has successfully completed building all the images in the
 
 It should be noted that `{{$names.company.lower}} push` is independent of git, so you are free to use any version control system you wish. This also means that it is possible to use [git submodules][git-submodules] in your project when deploying with `{{$names.company.lower}} push`.
 
-### Additional Options
-
-#### `--source, -s <source>`
-
-The `--source` flag allows you do define a path to your source code folder. This flag directs `push` to send that directory to be built on either the [cloud builder](#the-balenacloud-build-server) or [local device][local-mode]. You should ensure your folder follows the [standard {{$names.company.lower}} project structure](#project-structure).
-
-#### `--emulated, -e`
-
-The `--emulated` flag will force the [{{$names.cloud.lower}} builder](#the-balenacloud-build-server) to run an [qemu][qemu] emulated build. This means that your build will be executed on an `x86_64` CPU that emulates the target architecture of your application, rather than running on the native architecture of your device. You can see if the build is emulated in the first few lines of the builder output as below:
-
-```shell
-$ {{$names.company.lower}} push myApp --emulated
-[Info]     Starting build for myApp, user {{$names.company.lower}}_projects
-[Info]     Dashboard link: https://dashboard.balena-cloud.com/apps/1426783/devices
-[Info]     Running locally emulated build
-[Info]     Pulling previous images for caching purposes...
-[Success]  Successfully pulled cache images
-[main]     Step 1/2 : FROM {{$names.company.lower}}/secret_sauce
-[main]      ---> 6e48e49f10a6
-[main]     Step 2/2 : CMD cat /etc/os-release
-```
-
-__Note:__ The `--emulated` option is not available when pushing to a [localMode][local-mode] device.
-
-The emulated builds will also happen on the rare occasion that the native ARM builder is overloaded or unavailable.
-
-#### `--nocache, -c`
-
-The `--nocache` flag causes a fresh image build to take place, preventing the use of cached imaged layers from previous builds of this project. This is useful to
-ensure that the latest base image and packages are pulled. Note that the build logs may still display the message _"Pulling previous images for caching purposes,"_ because the cloud builder needs the previous images in order to prepare [delta updates](https://www.balena.io/docs/learn/deploy/delta/). When `--nocache` is used, the build logs will not display the "Using cache" lines for each build step of a Dockerfile.
+Note: Refer to the [`{{$names.company.lower}} push` command reference][cli-push-reference] for additional documentation.
```

Comment:
Doesn't your first suggestion now read:

> Refer to `balena push` command reference for additional information.

why drop the "the"? and is it just better style to use "information" vs. "documentation"?

Reply:
@robertgzr I agree with you there Robert, since we are specifically referring to one command which is `balena push` in this case. I think `the` should be there.